cli/crates/cli/src/main.rs

```diff
@@ -103,7 +110,22 @@ fn try_main() -> Result<(), CliError> {
         Some(("reset", _)) => reset(),
         Some(("login", _)) => login(),
         Some(("logout", _)) => logout(),
-        Some(("create", _)) => create(),
+        Some(("create", matches)) => {
+            let account_slug = matches.get_one::<String>("account").map(AsRef::as_ref);
```

Comment:
Do we really need these variables? It would also just work with something like:

```rust
let arguments = matches.get_one::<String>("account")
    .zip(matches.get_one::<String>("name"))
    .zip(matches.get_one::<String>("regions"))
    .map(...);
```

Reply:
Fair enough, fixed